packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/shared/base-target-group.ts

```diff
@@ -351,6 +352,21 @@ export abstract class TargetGroupBase extends Construct implements ITargetGroup
 
     return ret;
   }
+
+  protected validateHealthCheck(): string[] {
+    const ret = new Array<string>();
+
+    const intervalSeconds = this.healthCheck.interval?.toSeconds();
+    const timeoutSeconds = this.healthCheck.timeout?.toSeconds();
+
+    if (intervalSeconds && timeoutSeconds) {
+      if (intervalSeconds < timeoutSeconds) {
+        ret.push('Health check interval must be greater than the timeout; received interval ' +
```

Comment:
> must be greater than 

shouldn't the validation be `intervalSeconds <= timeoutSeconds` then? A link to the doc (in PR comment) stating this constraint will help.

Reply:
added a link to the doc. should be `<`.

Reply:
Thanks for the link.

> Timeout
> This value must be less than the Interval value.

So we shouldn't allow the case when `intervalSeconds == timeoutSeconds`. Then the condition should be `intervalSeconds <= timeoutSeconds`

Reply:
The PR introduced the validation https://github.com/aws/aws-cdk/pull/26031 also described the requirement as below:

> Ensure that HealthCheckTimeoutSeconds is not greater than HealthCheckIntervalSeconds.

So I guess it was just a mistake?

Ideally we should try the configuration (interval==timeout) both for ALB and NLB and it emits the same error.

Reply:
Take a look here: https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-elasticloadbalancingv2/test/nlb/target-group.test.ts#L386-L409. We have a specific test to not throw an error when they're the same. Confusing!

@pahud Is this something you could ask the AWS team about? It would be nice to have this added to the AWS documentation, and then we can decide which way the code needs to be.

Reply:
There was actually a similar discussion in the PR: https://github.com/aws/aws-cdk/pull/26031#discussion_r1237324560

So I guess we should keep the validation `intervalSeconds < timeoutSeconds`, and update the comment to e.g. `must be greater than or equal to`.

Reply:
I changed back to `<` and added a note. Thanks, @tmokmss 👍🏼